tests/test_elements/test_mol2any/test_mol2morgan_fingerprint.py

```diff
@@ -37,6 +37,26 @@ def test_can_be_constructed(self) -> None:
         for key, value in mol_fp.get_params().items():
             self.assertEqual(value, mol_fp_recreated.get_params()[key])
 
+    def test_counted_bits(self) -> None:
+        """Test if the option counted bits works as expected.
+
+        Returns
+        -------
+        None
+        """
+        mol_fp = MolToMorganFP(radius=2, n_bits=1024)
+        smi2mol = SmilesToMol()
+        pipeline = Pipeline(
+            [
+                ("smi2mol", smi2mol),
+                ("mol_fp", mol_fp),
+            ],
+        )
+        output_binary = pipeline.fit_transform(test_smiles)
+        pipeline.set_params(mol_fp__counted=True)
+        output_counted = pipeline.fit_transform(test_smiles)
+        self.assertTrue(np.all(output_counted.toarray() >= output_binary.toarray()))
```

Comment:
That's not a good test. Even when both are binary vectors, this would be true.

Maybe also check that the maximum in `ouput_counted` is >1? But I don't know if this is the case for all test molecules.

Reply:
Added the following tests
```python
self.assertTrue(
    np.all(np.flatnonzero(output_counted) == np.flatnonzero(output_binary))
)
self.assertTrue(np.any(output_counted > output_binary))
```

Reply:
Looks good :thumbsup: